ui/src/main/java/com/epam/eco/kafkamanager/ui/utils/UiUtils.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.eco.kafkamanager.ui.utils;
+/*******************************************************************************
+ *  Copyright 2023 EPAM Systems
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License.  You may obtain a copy
+ *  of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ *******************************************************************************/
+
+import static java.util.Objects.nonNull;
+
+/**
+ * @author Mikhail_Vershkov
+ */
+public class UiUtils {
+    private static final int TRUNCATE_FIELD_LENGTH = 127;
+    public static String getTruncatedDescription(String description) {
+        return nonNull(description) ?
```

Comment:
Let's check the description length on being less than 127 before performing substring

Reply:
I use here Math.min(TRUNCATE_FIELD_LENGTH,description.length() construction. So we haven't make substring from a value more than length.